projects/ffmpeg/build.sh

```diff
@@ -211,46 +263,86 @@ make tools/target_io_dem_fuzzer
 mv tools/target_io_dem_fuzzer $OUT/${fuzzer_name}
 
 #Build fuzzers for individual demuxers
-PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
-    --cc=$CC --cxx=$CXX --ld="$CXX $CXXFLAGS -std=c++11" \
-    --extra-cflags="-I$FFMPEG_DEPS_PATH/include" \
-    --extra-ldflags="-L$FFMPEG_DEPS_PATH/lib" \
-    --prefix="$FFMPEG_DEPS_PATH" \
-    --pkg-config-flags="--static" \
-    --enable-ossfuzz \
-    --libfuzzer=$LIB_FUZZING_ENGINE \
-    --optflags=-O1 \
-    --enable-gpl \
-    --enable-libxml2 \
-    --disable-muxers \
-    --disable-protocols \
-    --disable-devices \
-    --disable-shared \
-    --disable-encoders \
-    --disable-filters \
-    --disable-muxers  \
-    --disable-parsers  \
-    --disable-decoders  \
-    --disable-hwaccels  \
-    --disable-bsfs  \
-    --disable-vaapi  \
-    --disable-vdpau    \
-    --disable-crystalhd  \
-    --disable-v4l2_m2m  \
-    --disable-cuda_llvm  \
-    --enable-demuxers \
-    --disable-demuxer=rtp,rtsp,sdp \
+if [[ "$ARCHITECTURE" == i386 ]]; then
+	PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
+		--cc=$CC --cxx=$CXX --ld="$CXX $CXXFLAGS -std=c++11" \
+		--extra-cflags="-I$FFMPEG_DEPS_PATH/include" \
+		--extra-ldflags="-L$FFMPEG_DEPS_PATH/lib" \
+		--prefix="$FFMPEG_DEPS_PATH" \
+		--pkg-config-flags="--static" \
+		--enable-ossfuzz \
+		--libfuzzer=$LIB_FUZZING_ENGINE \
+		--optflags=-O1 \
+		--enable-gpl \
+		--enable-libxml2 \
+		--disable-muxers \
+		--disable-protocols \
+		--disable-devices \
+		--disable-shared \
+		--disable-encoders \
+		--disable-filters \
+		--disable-muxers \
+		--disable-parsers \
+		--disable-decoders \
+		--disable-hwaccels \
+		--disable-bsfs \
+		--disable-vaapi \
+		--disable-vdpau \
+		--disable-crystalhd \
+		--disable-v4l2_m2m \
+		--disable-cuda_llvm \
+		--enable-demuxers \
+		--disable-demuxer=rtp,rtsp,sdp \
+		--arch="i386" \
+		--cpu="i386" \
+		--disable-inline-asm \
+		--disable-asm \
+		--disable-neon \
+		;
+else
+	PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
+		--cc=$CC --cxx=$CXX --ld="$CXX $CXXFLAGS -std=c++11" \
+		--extra-cflags="-I$FFMPEG_DEPS_PATH/include" \
+		--extra-ldflags="-L$FFMPEG_DEPS_PATH/lib" \
+		--prefix="$FFMPEG_DEPS_PATH" \
+		--pkg-config-flags="--static" \
+		--enable-ossfuzz \
+		--libfuzzer=$LIB_FUZZING_ENGINE \
+		--optflags=-O1 \
+		--enable-gpl \
+		--enable-libxml2 \
+		--disable-muxers \
+		--disable-protocols \
+		--disable-devices \
+		--disable-shared \
+		--disable-encoders \
+		--disable-filters \
+		--disable-muxers \
+		--disable-parsers \
+		--disable-decoders \
+		--disable-hwaccels \
+		--disable-bsfs \
+		--disable-vaapi \
+		--disable-vdpau \
+		--disable-crystalhd \
+		--disable-v4l2_m2m \
+		--disable-cuda_llvm \
+		--enable-demuxers \
+		--disable-demuxer=rtp,rtsp,sdp \
+		;
+fi
```

Comment:
As with the other case, this should be represented in a nicer way so the common parts are not duplicated. That makes future maintaince also easier